src/main/java/view/NavigationMenu.java

```diff
@@ -0,0 +1,76 @@
+package view;
+
+import app.AppBuilder;
+import interface_adapter.login.LoginState;
+import interface_adapter.navigation.NavController;
+import interface_adapter.navigation.NavViewModel;
+
+import javax.swing.*;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import java.beans.PropertyChangeEvent;
+import java.beans.PropertyChangeListener;
+
+/**
+ * Navigation menu to the application
+ */
+public class NavigationMenu extends JMenuBar implements ActionListener {
+
+    private final JButton buttonSearch = new JButton("Search");
+    private final JButton buttonRecipeCollection = new JButton("Recipe Collection");
+    private final JButton buttonChangePassword = new JButton("Change Password");
+    private final JButton buttonLogout = new JButton("Logout");
+    private NavController navController;
+
+    public NavigationMenu(NavViewModel navViewModel) {
+        this.add(buttonSearch);
+        this.add(buttonRecipeCollection);
+        this.add(buttonChangePassword);
+        this.add(buttonLogout);
+        this.setVisible(true);
+
+
+//        /**
+//         * Invoked when an action occurs.
+//         *
+//         * @param e the event to be processed
+//         */
+//        @Override
+//        public void actionPerformed(ActionEvent e){
+//
+//        }
+//
+//        buttonSearch.addActionListener(
+//                new ActionListener() {
+//                    public void actionPerformed(ActionEvent e) {
+//                        navController.switchToSearchView();
+//                    }
+//                }
+//        );
+
+    }
```

Comment:
I agree with Emily's suggestion. Your coding style is clean and straightforward, so I’d recommend removing these lines if they’re not needed here or add/commit to keep up with your code flow.